app/src/org/commcare/utils/AndroidInstanceInitializer.java

```diff
@@ -39,7 +40,7 @@ public AndroidInstanceInitializer(CommCareSession session) {
     }
 
     public AndroidInstanceInitializer(CommCareSession session, UserSandbox sandbox, CommCarePlatform platform) {
-        super(session, sandbox, platform);
+        super(new SessionWrapper(session, platform, sandbox), sandbox, platform);
```

Comment:
this is not ideal as we are creating another object in constructor, but this was the most cleanest at this point. We should clean CommCareInatializer at some point to take `platform` and `sandbox` from Session Wrapper itself instead of passing repeated arguments.